script.js

```diff
@@ -1 +1,163 @@
-//CEOS 19기 프론트엔드 파이팅🔥 ദ്ദി˶ˊᵕˋ˵)
+document.addEventListener('DOMContentLoaded', function () {
+  let countSchedule = 0;
+  let countDone = 0;
+
+  // current schedule&done
+  function makeHistoryElement(id) {
+    if (id == 'schedule') {
+      countSchedule += 1;
+      if (countDone > 0) {
+        countDone -= 1;
+      }
+    } else if (id == 'done') {
+      countDone += 1;
+      if (countSchedule > 0) {
+        countSchedule -= 1;
+      }
+    } else if (id == 'newSchedule') {
+      countSchedule += 1;
+    } else if (id == 'trashSchedule') {
+      countSchedule -= 1;
+    } else if (id == 'trashDone') {
+      countDone -= 1;
+    }
```

Comment:
makeHistoryElement 함수 내에서 id 값에 따라 직접 countSchedule과 countDone을 조정하는 대신, 이를 별도의 함수(adjustCount)로 분리하여 중복을 제거하고 가독성을 높이는 방법도 좋은 것 같아요.

```suggestion
// ID에 따른 카운트 조정을 단순화할 수 있는 함수
function adjustCount(id) {
  switch(id) {
    case 'schedule':
      countSchedule++;
      break;
    case 'done':
      countDone++;
      break;
    case 'newSchedule':
      countSchedule++;
      break;
    case 'trashSchedule':
      countSchedule--;
      break;
    case 'trashDone':
      countDone--;
      break;
  }
}

```